astropy/stats/__init__.py

```diff
@@ -11,14 +11,35 @@
 
 """
 
+from . import funcs
 from .funcs import *  # noqa
```

Comment:
Do we still need the `*` imports now that we manually extend `__all__` below?

Reply:
yes, we still need all of these being available for direct imports, the list in `__all__` after all just a list of strings that's only been used for thr wild card imports